pom.xml

```diff
@@ -109,4 +109,44 @@
       </plugins>
     </pluginManagement>
   </build>
+  <profiles>
```

Comment:
Why do we need to redefine this profile here, as it is already defined in maven-parent-pom ?

Reply:
Explained  in this comment https://github.com/Meeds-io/social/pull/4067#discussion_r1778325696